internal/bminventory/inventory.go

```diff
@@ -354,6 +354,22 @@ func (b *bareMetalInventory) validateRegisterClusterInternalParams(params *insta
 	return nil
 }
 
+func (b *bareMetalInventory) validateAccessToARMFunctionality(ctx context.Context, cpuArchitecture string) error {
```

Comment:
can be more generic to support other archs validation, e.g. validateAccessToArchitectureSupport

Reply:
why? we do not currently limit access to any other arch.

Reply:
right, we only have a single additional arch now, but the intention is to support more archs later on as ppc.

Reply:
Done.